docs/.vuepress/enhanceApp.js

```diff
@@ -1,9 +1,9 @@
 export default ({ router }) => {
   router.addRoutes([
-    { path: '/main/spec/*', redirect: '/main/modules/' },
-    { path: '/main/spec/governance/', redirect: '/main/modules/gov/' },
-    { path: '/v0.41/', redirect: '/v0.42/' },
-    { path: '/v0.43/', redirect: '/v0.44/' },
-    { path: '/master/', redirect: '/main/' },
+    { path: '/main/spec/*', redirect: '/modules/' },
+    { path: '/main/spec/governance/', redirect: '/modules/gov/' },
+    { path: '/v0.41/', redirect: '/v0.42/' }, // TODO to fix
+    { path: '/v0.43/', redirect: '/v0.44/' }, // TODO to fix
+    { path: '/master/', redirect: '/' },
```

Comment:
Because we copy the last `index.html` and `404.html` to the root of the site, and we've generated the docs using `VUEPRESS_BASE=$path_prefix`, when we want to redirect to a link, it appends the `$path_prefix` to the redirection path.

Now that we are having `main` as default docs, we can use a workaround to have proper redirection, and we intend to redirect to a page within main docs.